scripts/reporting/batch_script_utils.py

```diff
@@ -0,0 +1,81 @@
+from subprocess import run
+
+
+def rehydrate_issue_body(body: str) -> dict:
```

Comment:
This is really kinda hard to follow - can you describe what it does and ideally provide an example?

Reply:
Yup, I broke it up into more readable code. It just turns the list of bulletpoints in the issue back into a dict.